Dockerfile

```diff
@@ -1,6 +1,14 @@
 # Build the manager binary
 FROM quay.io/operator-framework/helm-operator:v1.6.1
 
+LABEL name="Sumologic Kubernetes Collection Helm Operator"
```

Comment:
There is an OCI standard for the label names here: https://github.com/opencontainers/image-spec/blob/master/annotations.md#pre-defined-annotation-keys. I propose to use those.

Reply:
These labels should have these keys.

Reply:
I understand those come from the Red Hat requirements.

Reply:
yes